apps/system-e2e/global.d.ts

```diff
@@ -1,4 +1,6 @@
 // global.d.ts
+export {};
+
```

Comment:
What's this for?

Reply:
So that TS stops complaining about `.toHaveCountGreaterThan()`